.ci/bootstrap_catdog.sh

```diff
@@ -9,7 +9,7 @@ then
 else
     COMMIT_MSG="Initial commit\n\n[noissue]"
 fi
-echo $COMMIT_MSG
+echo "$COMMIT_MSG"
```

Comment:
If we want to be *really* correct, bash-params should be protected via ${PARAM}

Reply:
I *think* that is optional. This change is what made shellcheck silent. ;)
```suggestion
echo "${COMMIT_MSG}"
```